src/builder.rs

```diff
@@ -324,7 +324,7 @@ fn parse_evaluation_line<T>(line: T) -> LogDatum
 where
     T: AsRef<OsStr>,
 {
-    lazy_static! {
+    lazy_static::lazy_static! {
```

Comment:
My heuristic here was: if a macro is only ever used once, then call it directly and remove the `#[macro_use] extern crate` declaration.